config/application.rb

```diff
@@ -56,6 +56,10 @@ class Application < Rails::Application
         origins '*'
         resource '/evaluations/export*', headers: :any, methods: [:get]
       end
+      allow do
+        origins Settings.root_url
```

Comment:
Why are you using the `root_url` here? Why is it different from above? Why do we need to allow our own domain (the root_url) to access that resource (it's allowed by default)? Why is it working?